### PR TITLE
Implement create event endpoint

### LINKED ISSUE #14: Admin create event
## Task(s)
- [x] API route to create events
- [x] Add event to the database

## Acceptance Criteria

The admins can create events with custom name, start time, end time, points, secret code, join link, description

### PR DESCRIPTION
**Pull request by luiszugasti**
## Description

This is an open PR for fixing issue #14.
~~**Currently, the app server fails due to flask.cli.NoAppException** and I have no clue why this is.~~
I'm including it as a bug fix because os.environ was changed to [os.getenv](https://github.com/MLH-Fellowship/fellow-central/pull/19/commits/cb71e137f95f86784b4970ea20827d3aa389c039). << I'm actually not sure if thats correct anymore, but it looks like correct usage from [here.](https://pypi.org/project/python-dotenv/)
Also, there are new requirements that were added from the previous PR: that were not captured in requirements.txt, so the have been added.

Fixes #14

## Type of change

~~Please delete options that are not relevant.~~

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by luiszugasti** [comment]
Btw the Flask issue was resolved yesterday as discused, importing the FLASK_APP environment variable worked well.

**Comment by luiszugasti** [comment]
I am trying to test this in postman, but it appears to fail with SQL raising an error. Will follow up on it

File: backend/app.py [hunk 1]
```diff
@@ -147,6 +147,33 @@ def discord_callback():
...
+                          secret_code=secret_code_f,
+                          event_link=link)
+
+        db.session.add(new_event)
+        db.session.commit()
+
+        return f"Entry successfully posted."
+
+    else:
+        print("What")
```
Comment (cqvu) [inline backend/app.py:174]: lol do we want to keep this?

File: backend/app.py [hunk 2]
```diff
@@ -147,6 +147,33 @@ def discord_callback():
...
+        new_event = Event(name=event_name, start_time=start_time_f,
+                          end_time=end_time_f,
+                          points_amount=points,
+                          secret_code=secret_code_f,
+                          event_link=link)
+
+        db.session.add(new_event)
+        db.session.commit()
+
+        return f"Entry successfully posted."
```
Comment (cqvu) [inline backend/app.py:171]: Return as json?

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+.vscode/
```
Comment (cqvu) [inline .gitignore:1]: remove?

File: backend/app.py (hunk 1 above)
Comment (luiszugasti) [reply backend/app.py]: Nope, I removed it!

**Review by cqvu — APPROVED** [review]
Looks good 🚀

**Pull request merged**